build/docker/client/Dockerfile

```diff
@@ -1,12 +1,38 @@
-FROM redhat/ubi9-minimal AS users
+FROM redhat/ubi9 AS builder
 
-RUN microdnf install shadow-utils jq tar -y
+RUN dnf install -y shadow-utils wget jq tar https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
```

Comment:
Do we really need `wget` given that there is already `curl`, which is much more feature-rich?

Reply:
How is it supposed to be used?

Reply:
Good point. I'll remove it.